src/System.Management.Automation/help/UpdatableHelpCommandBase.cs

```diff
@@ -862,4 +872,19 @@ internal void ProcessException(string moduleName, string culture, Exception e)
 
         #endregion
     }
+
+     /// <summary>
+    /// Scope to which the help should be saved.
+    /// </summary>
+    public enum UpdateHelpScope
+    {
+        /// <summary>
+        /// Save the help content to the module directory. This is the default behavior.
```

Comment:
Personally, I don't like that we default to requiring elevation as it means that for most users they see an error and then need to rerun `Update-Help -Scope CurrentUser`.  Let's have this reviewed by @PowerShell/powershell-committee as it would be a breaking change.

Reply:
Updated.